packages/go/test/fixtures/10-go-mod-newer/README.md

```diff
@@ -0,0 +1,3 @@
+Build container has Go 1.18 and this test has a `go.mod` that says to use
```

Comment:
I don't believe the build container has Go installed at all. It should install and cache on the first deployment.

Reply:
Hmm, I believe you are correct. I'll fix this messaging.